### DISCUSSION TITLE
Teacher Verification (problems with location services)

### DISCUSSION THREAD

**Discussion by vsagraham** [community/community#65157] [GitHub Education]
### Hi everyone,

Hi,
First of all I know there are lots of similar tickets, but I haven't seen a solution to my problem.

I am trying and failing to get verified as a teacher because I keep getting asked to make sure location settings are enabled.
- I have tried with Chrome, Firefox, and Opera
- I have tried with a Chromebook, PC, Mac and Android phone
- I have enabled location settings in all browsers and devices
- I have tried using different networks, home, school and mobile phone hotspot

Any suggestions as to how I can proceed or speak to a human from Github to get this sorted would be wonderful.

Thanks in advance!

**Comment by arafajs** [comment]
Disable VPN.
Make sure that your ID location is the same as Your Device Location.

**Reply by vsagraham** [reply]
Thanks Abdula.
I tried with and without VPN. I also tried with different computers that don't actually have VPN installed.

**Reply by arafajs** [reply]
Make sure that you have enabled location services for all of the devices and browsers you are using. You can do this by going to your device's settings and finding the location option.

**Reply by vsagraham** [reply]
I did that. I did it on each devices settings (Mac, PC, Android) and on each browser's settings.
